# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pdksh/mips.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some hackery to get mips building.

  - Rene Rebe <rene@exactcode.de>

--- pdksh-5.2.14/sh.h.vanilla	2005-11-06 16:30:50.000000000 +0100
+++ pdksh-5.2.14/sh.h	2005-11-06 16:31:18.000000000 +0100
@@ -251,7 +251,7 @@
 # if SIZEOF_LONG >= 4
 #  define INT32	long
 # else /* SIZEOF_LONG */
-   #error cannot find 32 bit type...
+#  define INT32 int32_t
 # endif /* SIZEOF_LONG */
 #endif /* SIZEOF_INT */
 
